src/app/sponsors/page.tsx

```diff
@@ -0,0 +1,77 @@
+import Paragraph from '@/components/Paragraph';
+import Sponsors from '@/components/Sponsors';
+import Title from '@/components/Title';
+import type { SponsorType } from '@/data/sponsors';
+import Image from 'next/image';
+
+const SPONSOR_TYPE_COLORS = {
+    gold: '#FCC018',
+    silver: '#C3C3C3',
+    bronze: '#E8903F',
+} as const satisfies Record<SponsorType, string>;
+
+export default function SponsorsPage() {
+    return (
+        <div className="flex flex-col items-center gap-10">
+            <div className="flex gap-8">
+                <Image
+                    src="/images/yellow-duck.svg"
+                    alt="Duck"
+                    width={500}
+                    height={500}
+                    className="hidden w-20 md:block"
+                />
+                <Title colour="yellow">{`${new Date().getFullYear()} Sponsors`}</Title>
```

Comment:
Might be an issue with the `Title` component, but on md sized screens, the title's width isn't correct and it should be centred.
<img width="563" alt="image" src="https://github.com/compsci-adl/website/assets/47909638/c7b44022-9a21-4684-9952-d6721e258b36">